src/common/components/Button/index.styl

```diff
@@ -0,0 +1,12 @@
+@require 'cozy-ui'
+
+@media (max-width: (1023/basefont)rem)
```

Comment:
Don't we have variables or mixins for media queries? It feels weird  to have these values hard-coded here.

Reply:
We have these all over the place... Didn't knew we got mixins for that. But as we are more and more adverse to stylus, do we really want to go all-in with stylus mixins now?

Reply:
I absolutely think so. I don't share the adversion to stylus all and but regardless, duplicating magic numbers is a bad thing. We hopefully wouldn't do this without stylus either!